engine/language-server/src/main/scala/org/enso/languageserver/boot/MainModule.scala

```diff
@@ -196,7 +196,11 @@ class MainModule(serverConfig: LanguageServerConfig, logLevel: LogLevel) {
 
   lazy val bufferRegistry =
     system.actorOf(
-      BufferRegistry.props(fileManager, runtimeConnector),
+      BufferRegistry.props(
+        fileManager,
+        runtimeConnector,
+        TimingsConfig.default().withAutoSave(60.seconds)
```

Comment:
Isn't 60s a bit too much? I'd save in 10s...

Reply:
Agree, I'd even make it ~6 seconds

Reply:
Sure, np.